### PR TITLE
[New pattern] Lambda-Kinesis-Lambda js pattern

### PR DESCRIPTION
**Pull request by sabornibhattacharya**
*Issue #, if available:*

*Description of changes:*

This pattern talks about how to use Lambda functions with nodejs runtime to push and read records from a Kinesis Data Stream. This specifically demonstrates the implementation of serverless producer and consumer for a stream.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sabornibhattacharya** [comment]
I can't figure the actual cause of the failure [here](https://github.com/aws-samples/serverless-patterns/actions/runs/7194542293/job/19595287535?pr=1954)

File: lambda-kinesis-lambda-js/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "title": "Lambda to Kinesis Data Stream to Kinesis using SAM",
+  "description": "This sample project creates a REST API with Lambda proxy integration that invokes Bedrock-runtime API for text generation.",
+  "language": "Node.js",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This pattern talks about how to use Lambda functions with nodejs runtime to push and read records from a Kinesis Data Stream."
+      "This specifically demonstrates the implementation of serverless producer and consumer for a stream."
```
Comment (pputhran) [inline lambda-kinesis-lambda-js/example-pattern.json:11]: There is a comma missing here. You might want to validate using json validator

File: lambda-kinesis-lambda-js/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
+  "title": "Lambda to Kinesis Data Stream to Kinesis using SAM",
```
Comment (pputhran) [inline lambda-kinesis-lambda-js/example-pattern.json:2]: Please update this

File: lambda-kinesis-lambda-js/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "description": "This sample project creates a REST API with Lambda proxy integration that invokes Bedrock-runtime API for text generation.",
```
Comment (pputhran) [inline lambda-kinesis-lambda-js/example-pattern.json:3]: Description needs to be updated as well

**Review by pputhran — COMMENTED** [review]
Added a few comments

**Review by pputhran — APPROVED** [review]

**Pull request merged**